editor/animation_bezier_editor.h

```diff
@@ -98,6 +98,8 @@ class AnimationBezierTrackEdit : public Control {
 	void _menu_selected(int p_index);
 
 	void _play_position_draw();
+	bool _is_track_displayed(int track_index);
+	bool _is_track_curves_displayed(int track_index);
```

Comment:
```suggestion
	bool _is_track_displayed(int p_track_index);
	bool _is_track_curves_displayed(int p_track_index);
```
Prefix arguments with `p_`